doc/herbstluftwm.txt

```diff
@@ -478,12 +478,14 @@ bring 'WINID'::
 
 
 resize 'DIRECTION' ['FRACTIONDELTA']::
-    Changes the next fraction in specified 'DIRECTION' by 'FRACTIONDELTA' (which
-    defaults to 0.02 if none is supplied). 'DIRECTION' behaves as specified at
-    the 'focus' command. If a floating window is focused, it is resized and
-    grows into the specified 'DIRECTION'. In the parameter 'FRACTIONDELTA', you
-    should not omit the sign '-' or '+', because in future versions, the
-    behaviour may change if the sign is omitted. Example:
+    Changes the size of the focused frame (or floating window) in specified
```

Comment:
"(or floating window)" is a bit confusing here, because you first tell it's resized by `FRACTIONDELTA` and *then* change the behaviour expected for floating clients.

Reply:
Thanks, rephrased it a bit